test/github_organization_test.go

```diff
@@ -0,0 +1,43 @@
+package test
+
+import (
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"os"
+	"testing"
+)
+
+var githubOrganization, githubToken string
+
+func init() {
+	githubOrganization = os.Getenv("GITHUB_ORGANIZATION")
+	githubToken = os.Getenv("GITHUB_TOKEN")
+
+	if githubOrganization == "" {
+		panic("Please set a github organization using the GITHUB_ORGANIZATION environment variable.")
+	}
+
+	if githubToken == "" {
+		panic("Please set a github token using the GITHUB_TOKEN environment variable.")
+	}
+}
+
+func TestTerraformRepository(t *testing.T) {
+	t.Parallel()
+
+	terraformOptions := &terraform.Options{
+		// The path to where your Terraform code is located
+		TerraformDir: "../examples/organization",
+		Upgrade:      true,
+		// Variables to pass to our Terraform code using -var options
+		Vars: map[string]interface{}{
+			"github_token":        githubToken,
+			"github_organization": githubOrganization,
```

Comment:
is this needed as the github provider should pickup the environment anyway?

Reply:
good point! It is not necessarily needed!